swarm/pss/pss.go

```diff
@@ -325,6 +325,10 @@ func (self *Pss) handlePssMsg(msg interface{}) error {
 	pssmsg, ok := msg.(*PssMsg)
 
 	if ok {
+		if int64(pssmsg.Expire) < time.Now().Unix() {
```

Comment:
it is more golang style to handle the return on error case first like `if !ok {return err}`
but in this case i would even let `pss := msg.(*PssMsg)` panic since this would be an implementation bug

Reply:
But if it panics then someone could bring down the node by deliberately sending a wrong message struct on the protocol?